tests/python/integration/test_reduce.py

```diff
@@ -191,11 +191,11 @@ def test_rfactor_factor_axis():
     n = tvm.runtime.convert(1027)
     A = te.placeholder((n,), name="A")
     k = te.reduce_axis((0, n))
-    B = te.compute((1,), lambda i: te.sum(A[k], axis=k), name="B")
```

Comment:
how is the issue here? Why this pr expose this problem?

Reply:
The PR added extra check for the shape of the arguments for `np.testing.assert_allclose`